Ryabuhin Artiom/02.git.local/git_tree.txt

```diff
@@ -0,0 +1,24 @@
+*   c7fe392 (HEAD -> master) something went wrong ((
```

Comment:
Please provide separate output for each branch

Reply:
please tell me the command for output only for each branch.
I know how to show all commits only.